client/src/components/OverView/StarRating.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Would recommend putting "Star Rating" in the root components folder since it will be a shared component